apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -76,9 +82,26 @@ func (host *BareMetalHost) validateChanges(old *BareMetalHost) []error {
 		errs = append(errs, fmt.Errorf("bootMACAddress can not be changed once it is set"))
 	}
 
+	if err := validateAddStatusAnnotation(old, host); err != nil {
+		errs = append(errs, err)
+	}
+
 	return errs
 }
 
+func validateAddStatusAnnotation(old *BareMetalHost, new *BareMetalHost) error {
+
+	oldStatusNotEmpty := (old.Status != (BareMetalHostStatus{}))
```

Comment:
This is not the check we use in the controller: https://github.com/metal3-io/baremetal-operator/blob/main/controllers/metal3.io/baremetalhost_controller.go#L1676-L1678

Reply:
Thanks Zane, I've pushed a commit using the check used in the controller, will now run the CI test and see how it goes.